libs/utils/utils.js

```diff
@@ -959,6 +962,10 @@ async function documentPostSectionLoading(config) {
 
   const { default: delayed } = await import('../scripts/delayed.js');
   delayed([getConfig, getMetadata, loadScript, loadStyle, loadIms]);
+
+  import('../martech/analytics.js').then((analytics) => {
```

Comment:
Since this is in `documentPostSectionLoading`, analytics will not be run on any fragments that are loaded.

Reply:
Hmm.  It's working on homepage and we have a ton of fragments in there.
preview link: [https://analytics--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=mwpw-135672-analytics](https://analytics--homepage--adobecom.hlx.page/homepage/index-loggedout?milolibs=mwpw-135672-analytics)
source page: [https://adobe.sharepoint.com/:w:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7BE04B5FF7-CEAA-465C-B44E-539B8062B9F1%7D&file=index-loggedout.docx&action=default&mobileredirect=true](https://adobe.sharepoint.com/:w:/r/sites/adobecom/_layouts/15/Doc.aspx?sourcedoc=%7BE04B5FF7-CEAA-465C-B44E-539B8062B9F1%7D&file=index-loggedout.docx&action=default&mobileredirect=true)

But I'm totally willing to move it if there's a better location.  Where do you recommend?

Reply:
ok, I must be wrong there - if we encounter it later it can be fixed then.